### PR TITLE
Adds dataAtPath method to API

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: lib/Textile/API/ipfs.ts [hunk 1]
```diff
@@ -2,6 +2,11 @@ import { NativeModules } from 'react-native'
 
 const { TextileNode } = NativeModules
 
+export async function dataAtPath(path: string): Promise<string> {
+  const result = await TextileNode.dataAtPath(path)
+  return result as string
```
Comment (asutula) [inline lib/Textile/API/ipfs.ts]: `Buffer` (`UIntArray`) could be a better return type for "data".

Comment (asutula) [reply lib/Textile/API/ipfs.ts]: Or is the string returned a data url?

**Comment by andrewxhill** [comment]
yeah, data url

File: lib/Textile/API/ipfs.ts (hunk 1 above)
Comment (sanderpick) [reply lib/Textile/API/ipfs.ts]: The data could be anything at that path. So, no way to properly construct a data URL. It just returns a byte slice: `func (m *Mobile) DataAtPath(pth string) ([]byte, error)`

Comment (asutula) [reply lib/Textile/API/ipfs.ts]: Makes sense. This should return `Buffer` or `UIntArray` then.

Comment (andrewxhill) [reply lib/Textile/API/ipfs.ts]: ah okay, i just got lucky in that the string could just be dropped into the rn image uri

Comment (asutula) [reply lib/Textile/API/ipfs.ts]: Surprised that worked. Maybe RN `Image` has some default handling of string values that made it work.

**Pull request merged**